misc/tools.py

```diff
@@ -181,8 +179,8 @@ def checkmarks():
         return
 
     for k, v in functions:
-        print >>sys.stdout, "{:#x} : in function {:s}".format(k, func.name(func.byAddress(k)))
-        print >>sys.stdout, '\n'.join( ("- {:#x} : {:s}".format(a, m) for a, m in sorted(v)) )
+        six.print_("{:#x} : in function {:s}".format(k, func.name(func.byAddress(k))), file=sys.stdout)
```

Comment:
This call to `func.byAddress` should be using the non-aliased version, `func.by_address`, or `func.by`.